### DISCUSSION TITLE
Packages/com.unity.rendering.hybrid/manual/index

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#561] [Notes]
# Packages/com.unity.rendering.hybrid/manual/index

https://docs.unity3d.com/Packages/com.unity.rendering.hybrid@latest/index.html?subfolder=%2Fmanual%2Findex.html

**Comment by redwyre** [comment]
Link to new package: https://docs.unity3d.com/Packages/com.unity.entities.graphics@1.0/manual/index.html
